pkg/ccl/backupccl/backup_test.go

```diff
@@ -1303,34 +1303,75 @@ func checkInProgressBackupRestore(
 	var allowResponse chan struct{}
 	params := base.TestClusterArgs{}
 	knobs := base.TestingKnobs{
-		DistSQL: &execinfra.TestingKnobs{BackupRestoreTestingKnobs: &sql.BackupRestoreTestingKnobs{RunAfterProcessingRestoreSpanEntry: func(ctx context.Context) {
-			<-allowResponse
-		}}},
-		Store: &kvserver.StoreTestingKnobs{
-			TestingResponseFilter: func(ctx context.Context, ba roachpb.BatchRequest, br *roachpb.BatchResponse) *roachpb.Error {
-				for _, ru := range br.Responses {
-					switch ru.GetInner().(type) {
-					case *roachpb.ExportResponse:
-						<-allowResponse
-					}
-				}
-				return nil
-			},
-		}}
+		DistSQL: &execinfra.TestingKnobs{
```

Comment:
Can we delete or edit the comment above, now that we do not install a store response filter.